src/app/containers/App/index.test.jsx

```diff
@@ -1,38 +1,70 @@
-/*
- * © Jordan Tart https://github.com/jtart
- * https://github.com/jtart/react-universal-app
- */
 import React from 'react';
-import { shouldShallowMatchSnapshot } from '#testHelpers';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { StaticRouter, BrowserRouter } from 'react-router-dom';
 import { ClientApp, ServerApp } from '.';
+import * as App from './App';
+
+jest.mock('./App', () => jest.fn(() => <div>I am the App component</div>));
+
+jest.mock('react-router-dom', () => ({
+  BrowserRouter: jest.fn(),
+  StaticRouter: jest.fn(),
+}));
+BrowserRouter.mockImplementation(({ children }) => <div>{children}</div>);
```

Comment:
Does these things need to be mocked for the tests to work?

Reply:
I think the main purpose of mocking them is make it possible to `assert` that they were called with the correct props.

Reply:
OK cool. I'd separate out the assertion on line 34 into it's own test and give the tests better titles e.g. `App is called with correct props` and `ClientApp renders the App component` but I'd make sure the Browser router mock is removed before testing the latter. Then do the same for ServerApp